examples/pipeline/metrics.py

```diff
@@ -0,0 +1,28 @@
+from typing import Optional
+
+import torch
+
+
+def levenshtein_distance(r: str, h: str, device: Optional[str] = None):
```

Comment:
This seems worthwhile a separate PR already. Do you agree? In particular with the C++ extension we can create a JIT-able, fast version of this already.